src/vmm/src/vmm_config/machine_config.rs

```diff
@@ -254,13 +215,31 @@ pub enum CpuFeaturesTemplate {
     T2A,
 }
 
+/// Template types available for configuring the ARM CPU features that map
+/// to EC2 instances.
+#[cfg(target_arch = "aarch64")]
+#[derive(Clone, Copy, Debug, Deserialize, Eq, PartialEq, Serialize, Versionize)]
+pub enum CpuFeaturesTemplate {
+    // TODO REMOVE THIS WITH REFACTORING OF CpuFeaturesTemplate
+    /// Empty0
+    Empty0,
+    // TODO REMOVE THIS WITH REFACTORING OF CpuFeaturesTemplate
+    /// Empty1
+    Empty1,
```

Comment:
Are these still needed?

Reply:
So the reason for these dummy enum values is that for some reason when we restore snapshot on aarch64, the `cpu_template` enum is present in the snapshot (even thought it should not) and this enum has value None. Because of that, we try to deserialize this enum. And this deserialization fails, because now on aarch64 we only have 2 values in the enum. But if you look at the `CpuFeatureTemplate` for x86, the None value is 4th (so the integer value is 3). So I had to add 2 dummy values to make None variant to be 4th.